Bigtable/tests/Snippet/InstanceTest.php

```diff
@@ -39,6 +40,7 @@ class InstanceTest extends SnippetTestCase
     const INSTANCE_ID = 'my-instance';
     const INSTANCE_NAME = 'projects/my-awesome-project/instances/my-instance';
     const LOCATION_ID = 'us-east1-b';
+    const TABLE_NAME = 'projects/my-awesome-project/instances/my-instance/tables/my-table';
```

Comment:
Perhaps `INSTANCE_NAME . '/tables/my-table' for clarity?

Reply:
unfortunately that won't work in older versions of PHP.

Reply:
ack.